lessons/meta/installing-naucse/index.md

```diff
@@ -0,0 +1,71 @@
+# Lokální instalace Nauč se python
+
+K přidání kurzu nejdřív člověk potřebuje vlastní, lokální instalaci Nauč se.
+
+## Příprava
+
+První věc, kterou budeš potřebovat je Python, a to ve verzi 3.6.
+Pokud zrovna danou verzi Pythonu nainstalovanou nemáš, můžeš postupovat podle [návodu na instalaci Pythonu]({{lesson_url("beginners/install")}}).
+
+Druhá věc, kterou budeš potřebovat je git – pokud nemáš ten, můžeš postupovat podle [návodu na instalaci gitu]({{lesson_url("git/install")}}).
+
+Poslední věc, kterou potřebuješ už není žádný program, ale pár schopností.
+Je potřeba, aby jsi uměl{{a}} s terminálem a s gitem.
+Použivání příkazové řádky si můžeš připomenout v [návodu na používání terminálu]({{lesson_url("beginners/cmdline")}}) a používání gitu v [návodu na používání gitu]({{lesson_url("git/git-collaboration-2in1")}}).
+
+## Instalace
+
+Nejprve musíš naklonovat repozitář, ze kterého se Nauč se Python vykresluje.
+To uděláš tímto příkazem:
+
+```console
+$ git clone https://github.com/pyvec/naucse.python.cz
+```
+
+Poté přepni adresář do naklonovaného repozitáře:
+
+```console
+$ cd naucse.python.cz
+```
+
+A vytvoři si v něm virtuální prostředí – pokud nevíš jak na to, můžeš se podívat do návodu na instalaci Pythonu, na který je odkaz výše.
+Virtuální prostředí si rovnou aktivuj.
+
+Poslední krok instalace je nainstalování závislostí:
+
+```console
+(__venv__)$ python -m pip install -r requirements.txt
+```
+
+{{ anchor('launch') }}
+## Spuštění
+
+Teď když máš Nauč se nainstalované, tak stačí už jen pustit.
+Nauč se jde pustit ve dvou režimech.
+První režim vykresluje každou stránku pokaždé znova – hodí se na vývoj, aby byly všechny změny okamžitě vidět.
+Pustí se následovně:
+
+```console
+(__venv__)$ PYTHONPATH=. python -m naucse serve
+ * Running on http://0.0.0.0:8003/ (Press CTRL+C to quit)
+ * Restarting with stat
+ * Debugger is active!
+ * Debugger PIN: 256-270-314
+```
```

Comment:
Funguje nastavování proměnné prostředí takhle i na Windows?

Reply:
Nefunguje, přidal jsem nad to separátně nastavování proměné